lib/tools.js

```diff
@@ -177,18 +177,18 @@ function asChindings (instance, bindings) {
 function getPrettyStream (opts, prettifier, dest, instance) {
   if (prettifier && typeof prettifier === 'function') {
```

Comment:
Another option/ Maybe we can emit warning here once on stream creation?
Something like 
```
      process.emitWarning(
        'PrettyPrint does not support flushing',
      )
```

Reply:
How could you detect if the user intend to call flushSync?

Reply:
Fair remark, but isn't it more explicit to warn user during "application" start, instead of when problem is stumbled upon. 
Moreover `warning` can be missing from logs, cause application can exit before something has been written?

Reply:
Sorry for silly questions/remarks I don't have much experience in production js (moreover nodejs) code, and just want to acquire understanding

Reply:
> Fair remark, but isn't it more explicit to warn user during "application" start, instead of when problem is stumbled upon.

How can you detect that? This is triggered by folks manually calling `pino.stream.flushSync()` or are you triggering it in some other way?

Reply:
I meant without detection, just for every `pretty stream` user, for them to know that it's impossible to use `flushSync`. 
I know that this is written in docs, but warning still exist for some reason and is shown only once.
But now I see that this may be too disturbing.

> are you triggering it in some other way?
It's triggered by `logger.fatal` in our case